src/js/index.js

```diff
@@ -0,0 +1,8 @@
+import ReactDOM from 'react-dom';
+import React from 'react';
+import Component from '~/Component';
```

Comment:
Added a plugin so that we can use relative paths.  IMO, this is better than having '../../Component' and './Component' everywhere